app/views/wiki/show.html.erb

```diff
@@ -29,7 +29,6 @@
     <%= render :partial => "sidebar/featured" %>
   <% else %>
     <%= render :partial => "sidebar/related" %>
-  <% end %>
```

Comment:
Oops, actually I think we'll need the extra "end" because of the line above we want to re-add. Would you mind re-adding what used to be line 28? Thanks!

Reply:
Hm. Not following. There's the if above that has to do with time that has an end. And then there's our unless related to the sidebar that has an end. Are you saying our unless needs a second end?

Reply:
Sorry! I meant that the final version should look like:

```ruby
<% unless @node.has_tag('sidebar:none') %>
  <% if (@wikis.nil? && @notes.nil?) || @node.has_tag('sidebar:featured') %>
    <%= render :partial => "sidebar/featured" %>
  <% else %>
    <%= render :partial => "sidebar/related" %>
  <% end %>
<% end %>
```

I think i miswrote the original because the `unless` needs an `end` and so does the `if` -- so there should be two at the end. Sorry about that!

Reply:
Ah. I'm with you now. Cool I'll submit PR for that soon.